integrationtest/initial_release/data/mail.txt

```diff
@@ -1,5 +1,5 @@
 First line is skipped
-Subject: [MajorRelease] sandbox v1.0.0
+Subject: [MajorRelease] initial_release v1.0.0
```

Comment:
BTW, why this depends on the location of the test? I would avoid that if possible.